lib/wings/active_fedora_converter.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Wings
+  class ActiveFedoraConverter
+    ##
+    # @!attribute [rw] resource
+    #   @return [Valkyrie::Resource]
+    attr_accessor :resource
+
+    ##
+    # @param [Valkyrie::Resource]
+    def initialize(resource:)
+      @resource = resource
+    end
+
+    ##
+    # @return [Hash<Symbol, Object>]
+    def attributes
+      attrs = resource.attributes.reject { |k, _| k.to_s.end_with? '_ids' }
```

Comment:
I feel like it would be worth commenting the intention here and its near term removal.

Reply:
:+1: